Products/ZenEvents/tests/test_zeneventd.py

```diff
@@ -39,14 +40,14 @@ def setUp(self):
             created_time=1523044529575,
             event_class="/ZenossRM",
             actor=EventActor(
-                element_type_id=10,
+                element_type_id=ModelElementType.DEVICE,
```

Comment:
`element_type_id` is field of the enumerated type `ModelElementType`.  The value 10 is not a valid enumeration of `ModelElementType`.  This code should have never worked.